src/applications/vaos/services/patient/index.js

```diff
@@ -294,10 +312,25 @@ export async function fetchFlowEligibilityAndClinics({
     requestReasons: [],
   };
 
+  // Call not added above if removeFacilityConfigCheck is true, but requires resolved eligibility status to determine if needed
+  if (
+    typeOfCareRequiresPastHistory &&
+    removeFacilityConfigCheck &&
+    results.patientEligibility.direct
```

Comment:
A direct value would never be true from eligibility if directSchedulingEnabled is false (because it would not be requested)